* auto-gen TOC:
{:toc}

### PrimeKG: A Knowledge Graph for Precision Medicine.
PrimeKG integrates 20 high-quality resources to describe 17,080 diseases with 4,050,249 relationships representing ten major biological scales, including disease-associated protein perturbations, biological processes and pathways, anatomical and phenotypic scales, and the entire range of approved drugs with their therapeutic action, considerably expanding previous efforts in disease-rooted knowledge graphs. PrimeKG contains an abundance of 'indications', 'contradictions', and 'off-label use' drug-disease edges that lack in other knowledge graphs and can support AI analyses of how drugs affect disease-associated networks.

- [Paper](https://www.nature.com/articles/s41597-023-01960-3)

- [Code](https://github.com/mims-harvard/PrimeKG)

- [Data](https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/IXA7BM)

- [Website](https://zitniklab.hms.harvard.edu/projects/PrimeKG/)

### PreMedKB: an integrated precision medicine knowledgebase for interpreting relationships between diseases, genes, variants and drugs
Precision Medicine Knowledgebase (PreMedKB) integrate the four fundamental components of precision medicine: diseases, genes, variants and drugs. PreMedKB allows for search of comprehensive information within each of the four components, the relationships between any two or more components, and importantly, the interpretation of the clinical meanings of a patient's genetic variants. PreMedKB is an efficient and user-friendly tool to assist researchers, clinicians or patients in interpreting a patient's genetic profile in terms of discovering potential pathogenic variants, recommending therapeutic regimens, designing panels for genetic testing kits, and matching patients for clinical trials.

- [Paper](https://pubmed.ncbi.nlm.nih.gov/30407536/)

- [Code]()

- [Website]()

### Orphanet Data

Orphanet is a unique resource, gathering and improving knowledge on rare diseases so as to improve the diagnosis, care and treatment of patients with rare diseases. Orphanet aims to provide high-quality information on rare diseases, and ensure equal access to knowledge for all stakeholders. Orphanet also maintains the Orphanet rare disease nomenclature (ORPHAcode), essential in improving the visibility of rare diseases in health and research information systems.

Orphanet was established in France by the INSERM (French National Institute for Health and Medical Research) in 1997.This initiative became a European endeavour from 2000, supported by grants from the European Commission: Orphanet has gradually grown to a Consortium of 40 countries, within Europe and across the globe.

- [Paper]()

- [Code]()

- [Website](https://www.orphadata.com/)

### Human Symptoms Disease Network (HSDN)

Use large-scale medical bibliographic records and the related Medical Subject Headings (MeSH) metadata from PubMed, to generate a symptom-based network of human diseases (Human Symptoms Disease Network, HSDN), where the link weight between two diseases quantifies the similarity of their respective symptoms. By integrating disease–gene association and protein–protein interaction (PPI) data, we investigate the correlations between the symptom similarity of diseases and their degree of shared genes or PPIs.

- [Paper](https://www.nature.com/articles/ncomms5212)

- [Code](https://github.com/dhimmel/hsdn)

- [Data](https://www.nature.com/articles/ncomms5212#MOESM1042)


### SymMap: an integrative database of traditional Chinese medicine enhanced by symptom mapping

The pharmaceutical industry has heavily emphasized phenotypic drug discovery (PDD), which relies primarily on knowledge about phenotype changes associated with diseases. Traditional Chinese medicine (TCM) provides a massive amount of information on natural products and the clinical symptoms they are used to treat, which are the observable disease phenotypes that are crucial for clinical diagnosis and treatment. Curating knowledge of TCM symptoms and their relationships to herbs and diseases will provide both candidate leads and screening directions for evidence-based PDD programs. 

SymMap is an integrative database of traditional Chinese medicine enhanced by symptom mapping. It contains 1717 TCM symptoms and related them to 499 herbs and 961 symptoms used in modern medicine based on a committee of 17 leading experts practicing TCM, 5235 diseases associated with these symptoms, 19 595 herbal constituents (ingredients) and 4302 target genes, and built a large heterogeneous network containing all of these components. Thus, SymMap integrates TCM with modern medicine in common aspects at both the phenotypic and molecular levels.

- [Paper](https://academic.oup.com/nar/article/47/D1/D1110/5150228)

- [Code]()

- [Website](http://www.symmap.org/)

- [Data]()

### MetSigDis: a manually curated resource for the metabolic signatures of diseases

MetSigDis, a manually curated resource, aims to provide a comprehensive resource of metabolite alterations in various diseases. MetSigDis provides a search engine to query metabolic signatures and diseases along with relationships among them. Furthermore, to help users interactively analyze associations among diseases in metabolite level online, MetSigDis provides a network visualization tool to show associations among diseases and metabolites.↳

The current version of MetSigDis documents 6,849 curated relationships between 2,420 metabolites and 129 diseases across 8 species involving Homo sapiens, Rat, Mouse, Drosophila melanogaster, Triatomine, Mice, Pig, and Mus musculus. Each entry in the MetSigDis contains detailed information on a relationship between metabolite and disease, including metabolite ID, metabolite name, disease ID, disease name, species, metabolomics analytical platforms, tissue, metabolite alteration, and literature reference.

- [Paper](https://academic.oup.com/bib/article/20/1/203/4091292)

- [Code]()

- [Website](http://bio-annotation.cn/MetSigDis/)

- [Data](http://bio-annotation.cn/MetSigDis/download)

### gutMGene: a comprehensive database for target genes of gut microbes and microbial metabolites

Metagenomic sequencing of fecal samples has identified 3.3×106 non-redundant microbial genes from up to 1,500 different species. One of the contributions of gut microbiota to host biology is the circulating pool of bacterially derived small-molecule metabolites. It has been estimated that 10% of metabolites found in mammalian blood are derived from the gut microbiota, where they can produce systemic effects on the host through activating or inhibiting gene expression. Currently, detailed information on target genes of gut microbes and microbial metabolites is scattered in literatures and no online repository is available for collecting these data.

Here, gutMGene team manually extracted microbe-metabolite, microbe-target, and metabolite-target relationships in Human and Mouse from almost 400 publications. All of these relationships were experimentally validated in vivo or in vitro and measured by RT-qPCR, high performance liquid chromatography, 16S rRNA sequence, and so on.

- [Paper](https://academic.oup.com/nar/article/50/D1/D795/6368055)

- [Code]()

- [Website](http://bio-annotation.cn/gutmgene/home.dhtml)

- [Data](http://bio-annotation.cn/gutmgene/resource.dhtml)


### gutMDisorder: a comprehensive database for dysbiosis of gut microbiota in phenotypes and interventions

Gut microbiota plays a significant role in maintaining host health, and conversely, disorders potentially lead to dysbiosis, an imbalance in the composition of the gut microbial community. Intervention approaches, such as medications, diets, and several others, also alter the gut microbiota in either a beneficial or harmful direction.

A comprehensive resource for associations between gut microbes and phenotypes or interventions in Human and Mouse, derived from manual literature extraction and raw data reprocessing.

- [Paper](https://academic.oup.com/nar/article/51/D1/D717/6754909)

- [Code]()

- [Website](http://bio-annotation.cn/gutMDisorder/)

- [Data](http://bio-annotation.cn/gutMDisorder/resource.dhtml)


### OAHG: An integrated resource for annotating human genes with multi-level ontologies

In recent years, diverse products of genes in transcriptional level were well investigated, such as protein-coding gene (PCG), miRNA, lncRNA, circRNA, and so on. Their roles that vary from molecule to phenotype in human body attracted much attention these years. Now the annotation of genes’ roles using normalized terminologies is urgent for quantitative analyses. To this end, multiple manually curated databases were established about the annotation of an individual type of roles for an individual type of gene, such as miRNA-disease annotation (HMDD), lncRNA-disease annotation (LncRNADisease), circRNA-disease annotation (CircRNA2Disease), GOA, and so on. Although GeneRIF documented multiple roles of genes in the ‘functional description’ section, these roles were not normalized to ontologies. In summary, there is no online repository for annotating biological roles of genes comprehensively using ontogoies except for Ontogene.

Updated version at: [OntoGene: a resource for annotating human genes at transcriptional level with ontologies](http://bio-annotation.cn/OntoGene/)

- [Paper](https://www.nature.com/articles/srep34820)

- [Code]()

- [Website](http://bio-annotation.cn/Ontogene/Home.jsp)

- [Data](http://bio-annotation.cn/Ontogene/Download.jsp)


### FORUM: building a Knowledge Graph from public databases and scientific literature to extract associations between chemicals and diseases

Metabolomics studies aim at reporting a metabolic signature (list of metabolites) related to a particular experimental condition. These signatures are instrumental in the identification of biomarkers or classification of individuals, however their biological and physiological interpretation remains a challenge. 

FORUM is an open knowledge network aiming at supporting metabolomics results interpretation in biomedical sciences and automated reasoning. Containing more than 8 billion statements, it federate data from several life-science resources such as PubMed, ChEBI and PubChem. Leveraging the bridges in this linked dataset, we automatically extract associations between compound and biomedical concepts, using literature metadata enrichment.

- [Paper](https://academic.oup.com/nar/article/49/D1/D605/5910787)

- [Code](https://github.com/eMetaboHUB/Forum-DiseasesChem)

- [Website](https://forum-webapp.semantic-metabolomics.fr/)


### DISNET: A framework for extracting phenotypic disease information from public sources

Within the global endeavour of improving population health, one major challenge is the identification and integration of medical knowledge spread through several information sources. The creation of a comprehensive dataset of diseases and their clinical manifestations based on information from public sources is an interesting approach that allows one not only to complement and merge medical knowledge but also to increase it and thereby to interconnect existing data and analyse and relate diseases to each other. 

The DISNET database integrates phenotypic and genetic-biological characteristics of diseases and information on drugs from several sources curated by experts and unstructured textual sources. For phenotypic characteristics, information is retrieved from Wikipedia, PubMed and MayoClinic and more are being added; for biological characteristics, information is retrieved from DisGeNet to obtain gene and protein data, WikiPathways to obtain metabolic pathways data and IntAct to obtain protein interaction data; and for drug information, PharmGKB is used, among others.

- [Paper](https://peerj.com/articles/8580/)

- [Code]()

- [Website](http://disnet.ctb.upm.es/)

- [Data](https://disnet.ctb.upm.es/about/database)