### BioPortal: The National Center for Biomedical Ontology (NCBO) BioPortal

BioPortal provides access to commonly used biomedical ontologies and to tools for working with them. BioPortal allows you to: browse the library of ontologies; search for a term across multiple ontologies; browse mappings between terms in different ontologies; receive recommendations on which ontologies are most relevant for a corpus; annotate text with terms from ontologies; search biomedical resources for a term; browse a selection of projects that use BioPortal resources.

- [Paper]()

- [Code](https://github.com/ncbo)

- [Website](https://bioportal.bioontology.org/)

### OxO: EMBL-EBI Ontology Xref Service (OxO)

OxO is a service for finding mappings (or cross-references) between terms from ontologies, vocabularies and coding standards. OxO imports mappings from a variety of sources including the Ontology Lookup Service and a subset of mappings provided by the UMLS.

- [Paper]()

- [Code](https://github.com/EBISPOT/OxO)

- [Website](https://www.ebi.ac.uk/spot/oxo/index)

### Oncobee

Ontobee: A linked data server designed for ontologies. Ontobee is aimed to facilitate ontology data sharing, visualization, query, integration, and analysis. Ontobee dynamically dereferences and presents individual ontology term URIs to (i) HTML web pages for user-friendly web browsing and navigation, and to (ii) RDF source code for Semantic Web applications. Ontobee is the default linked data server for most OBO Foundry library ontologies. Ontobee has also been used for many non-OBO ontologies.

- [Paper]()

- [Code]()

- [Website](http://www.ontobee.org/)

### OBO Foundry: Symptom Ontology

The symptom ontology was designed around the guiding concept of a symptom being: “A perceived change in function, sensation or appearance reported by a patient indicative of a disease”. Understanding the close relationship of Signs and Symptoms, where Signs are the objective observation of an illness, the Symptom Ontology will work to broaden it’s scope to capture and document in a more robust manor these two sets of terms. Understanding that at times, the same term may be both a Sign and a Symptom.

- [Paper]()

- [Code](https://github.com/DiseaseOntology/SymptomOntology)

- [Website](https://obofoundry.org/ontology/symp.html)

- Related Links:

    - [BioPotal](https://bioportal.bioontology.org/ontologies/SYMP)

### Orphanet Rare Disease ontology (ORDO)

The Orphanet Rare Disease ontology (ORDO) was initially jointly developed by Orphanet and the EBI to provide a structured vocabulary for rare diseases capturing relationships between diseases, genes and other relevant features which will form a useful resource for the computational analysis of rare diseases. It is derived from the Orphanet database www.orpha.net , a multilingual database dedicated to rare diseases populated from literature and validated by international experts. It integrates a nosology (classification of rare diseases), relationships (gene-disease relations, epidemiological data) and connections with other terminologies (MeSH, UMLS, MedDRA), databases (OMIM, UniProtKB, HGNC, ensembl, Reactome, IUPHAR, Genatlas) or classifications (ICD-10).
The ontology will be maintained by Orphanet and further populated with new data. Orphanet classifications can be browsed in the OLS view. The Orphanet Rare Disease Ontology is updated every six months and follows the OBO guidelines on deprecation of terms.

- [Paper]()

- [Code]()

- [Website](https://www.orphadata.com/ordo/)


### PhenCards 

PhenCards is a database and web server intended as a one-stop shop for previously disconnected biomedical knowledge related to human clinical phenotypes. Users can query human phenotype terms or clinical notes. PhenCards obtains relevant disease/phenotype prevalence and co-occurrence, drug, procedural, pathway, literature, grant, and collaborator data. PhenCards recommends the most probable genetic diseases and candidate genes based on phenotype terms from clinical notes. PhenCards facilitates exploration of phenotype, e.g., which drugs cause or are prescribed for patient symptoms, which genes likely cause specific symptoms, and which comorbidities co-occur with phenotypes.

- [Paper](https://doi.org/10.1186/s13073-021-00909-8)

- [Code]()

- [Website](https://phencards.org/)


### The Human Phenotype Ontology (HPO)

The Human Phenotype Ontology (HPO) project provides an ontology of medically relevant phenotypes, disease-phenotype annotations, and the algorithms that operate on these. The HPO can be used to support differential diagnostics, translational research, and a number of applications in computational biology by providing the means to compute over the clinical phenotype. The HPO is being used for computational deep phenotyping and precision medicine as well as integration of clinical data into translational research. Deep phenotyping can be defined as the precise and comprehensive analysis of phenotypic abnormalities in which the individual components of the phenotype are observed and described. The HPO is being increasingly adopted as a standard for phenotypic abnormalities by diverse groups such as international rare disease organizations, registries, clinical labs, biomedical resources, and clinical software tools and will thereby contribute toward nascent efforts at global data exchange for identifying disease etiologies (Köhler et al, 2017).

- [Paper]()

- [Code]()

- [Website](https://hpo.jax.org/app/)


### Symptom Annotation Made Simple (SAMS)

Precision medicine needs precise phenotypes. The Human Phenotype Ontology (HPO) uses clinical signs instead of diagnoses and has become the standard annotation for patients’ phenotypes when describing single gene disorders. Use of the HPO beyond human genetics is however still limited. With SAMS (Symptom Annotation Made Simple), we want to bring sign-based phenotyping to routine clinical care, to hospital patients as well as to outpatients. Our web-based application provides access to three widely used annotation systems: HPO, OMIM, Orphanet. Whilst data can be stored in our database, phenotypes can also be imported and exported as Global Alliance for Genomics and Health (GA4GH) Phenopackets without using the database. The web interface can easily be integrated into local databases, e.g. clinical information systems. SAMS offers users to share their data with others, empowering patients to record their own signs and symptoms (or those of their children) and thus provide their doctors with additional information. We think that our approach will lead to better characterised patients which is not only helpful for finding disease mutations but also to better understand the pathophysiology of diseases and to recruit patients for studies and clinical trials.

- [Paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9252818/)

- [Code]()

- [Website](https://www.genecascade.org/SAMS/)


### Online Mendelian Inheritance in Man (OMIM)

OMIM is a comprehensive, authoritative compendium of human genes and genetic phenotypes that is freely available and updated daily. The full-text, referenced overviews in OMIM contain information on all known mendelian disorders and over 16,000 genes. OMIM focuses on the relationship between phenotype and genotype. It is updated daily, and the entries contain copious links to other genetics resources.

This database was initiated in the early 1960s by Dr. Victor A. McKusick as a catalog of mendelian traits and disorders, entitled Mendelian Inheritance in Man (MIM). Twelve book editions of MIM were published between 1966 and 1998. The online version, OMIM, was created in 1985 by a collaboration between the National Library of Medicine and the William H. Welch Medical Library at Johns Hopkins. It was made generally available on the internet starting in 1987. In 1995, OMIM was developed for the World Wide Web by NCBI, the National Center for Biotechnology Information.

- [Paper]()

- [Code]()

- [Website](https://www.omim.org/)

- Related Links:

    - [Seminar - Slides](http://www.cs.rpi.edu/~szymansk/fns.21/slides/20_Seminar_Vijay_Sadashivaiah_Human_Disease_Network_Final.pdf)
    